test/fixtures/win.cjs

```diff
@@ -457,6 +457,10 @@ module.exports.escape = {
         input: "a@b@c",
         expected: { interpolation: "a@b@c", noInterpolation: "a@b@c" },
       },
+      {
+        input: "a @b",
+        expected: { interpolation: "a `@b", noInterpolation: "a @b" },
+      },
```

Comment:
Is escaping (of this and all other characters covered by this Pull Request) also necessary after other whitespace characters? Whether "yes" or "no", there should be test cases for those other whitespace characters as well.